### PR TITLE
Module name change

### PR DESCRIPTION
**Pull request by bendvc**
Salesforce recommended a name change to align with their branding.

Status: **Opened for visibility**

Reviewers: @kevinxh @jeremywiebe 

## Changes
- Update readme, changelog
- Reset version number to 0.0.1 for new npm module
- Rebuild docs
- Update ascii art

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
-## v0.2.2 (November 3, 2017)
```
Comment (jeremywiebe) [inline CHANGELOG.md]: Sneaky. :P

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-[![NPM](https://nodei.co/npm/ocapi-client.png?downloads=true&stars=true)](https://nodei.co/npm/ocapi-client/)
+[![NPM](https://nodei.co/npm/commercecloud-ocapi-client.png?downloads=true&stars=true)](https://nodei.co/npm/commercecloud-ocapi-client/)
 
-[![CircleCI](https://circleci.com/gh/mobify/ocapi-client.svg?style=svg)](https://circleci.com/gh/mobify/ocapi-client)
+[![CircleCI](https://circleci.com/gh/mobify/commercecloud-ocapi-client.svg?style=svg)](https://circleci.com/gh/mobify/commercecloud-ocapi-client)
 
 ## Introduction
 
-ShopApi - JavaScript client for Salesforce OCAPI shop api.
+ShopApi - JavaScript client for CommerceCloud OCAPI shop api.
```
Comment (jeremywiebe) [inline README.md]: I'd still keep "Salesforce" in there... it's a mouthful but the full name is "Salesforce Commerce Cloud"

image

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ocapi-client",
-  "version": "0.2.2",
+  "name": "commercecloud-ocapi-client",
+  "version": "0.0.1",
   "description": "An ES6 JavaScript Client for Salesforce Open Commerce API",
```
Comment (jeremywiebe) [inline package.json]: So is the `lib/index.js` transpiled to ES5 or do we ship the library as ES6?

File: README.md (hunk 2 above)
Comment (bendvc) [reply README.md]: 👍

File: package.json (hunk 3 above)
Comment (bendvc) [reply package.json]: Done and done

File: package.json [hunk 4]
```diff
@@ -47,7 +47,6 @@
     "mobify-code-style": "2.8.1",
     "mocha": "~2.3.4",
     "rollup": "^0.49.3",
-    "rollup-plugin-babel": "^3.0.2",
     "rollup-plugin-uglify": "^2.0.1",
```
Comment (jeremywiebe) [inline package.json]: We can drop the uglify plugin too, then, right?

Comment (bendvc) [reply package.json]: 👍

**Pull request merged**